azure-container-apps/ReadMe.md

```diff
@@ -1,337 +1,349 @@
-    2. Change the **Ingress Traffic** to Accepting traffic from anywhere.
-    3. Specify the **Target Port** as **3002.**
-8. Select **Review + create.**
-9. Select **Create.**
-10. Once the deployment is complete, select **Go to Resource**.
-
-> **Note:** Expect the deployment to fail as it is expecting the second container to deploy successfully.
-
-### Step 3: Finalize your SCIM bridge deployment to use your scimsession secret and add the Redis Container.
-
-1. Within Azure Container App browser tab, Select **Secrets** along the left hand side.
-    1. Select **Add**.
-    2. Enter **scimsession** as the **Key field**.
-    3. Enter the base64 value into the **Value** field. (This is what we added to your 1Password **scimsession file item** in the getting started section)
-    4. Select **Add**.
-    5. Allow the secret to finish creating.
-2. Select **Containers** along the left hand side.
-    1. Select **Edit and deploy** along the top.
-    2. Put a check in the box next to your **op-scim-bridge** container and select **Edit**.
-    3. Scroll down to the Environment variables and edit the **OP_SESSION** variable to have a **Source** of **Reference a secret** and for the **Value**, select the **scimsession** secret.
-    4. Select **Save**.
-    5. Select the **+ Add** button, selecting **App container**.
-    6. Change the name of the container to **op-scim-redis**.
-    7. Select **Docker Hub** for the image source.
-    8. Enter **redis** for the **Image and Tag**.
-    9. Change the CPU and Memory to 0.25 CPU cores, 0.5 Gi memory.
-    10. Add the following **Environment variables**:
-        1. `REDIS_ARGS` with value of `--maxmemory 256mb --maxmemory-policy volatile-lru`.
-    11. Select **Add**.
-    12. Select **Next : Scale >**.
-    13. Drag both circles so the min and max are both set to 1.
-    14. Select **Create**.
-    15. Wait for the notification that the revision deployment was successful.
-    16. Select **Overview**.
-    17. On the right hand side, click on the Application Url (this is your SCIM bridge URL), to access and test your SCIM bridge.
-    18. You should be prompted to log into the SCIM bridge with your `bearer token`.
-
-### Step 4: Follow the steps to connect your Identity provider to the SCIM bridge.
- - [Connect your Identity Provider](https://support.1password.com/scim/#step-3-connect-your-identity-provider)
+
+3. Open the Azure Portal and go to the [Container Apps](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.App%2FcontainerApps) page.
+
+4. Choose **Secrets** from the Settings section in the sidebar.
+
+5. Click **Add**, the fill out the following fields:
+    1. **Key**: Enter `workspace-credentials`.
+    2. **Value**: Paste the base64 value you just created.
+
+6. Click **Add**, then wait until the secret is created.
+
+7. Add two environment variables to the new secret:
+    1. `OP_WORKSPACE_CREDENTIALS`
+    2. `OP_WORKSPACE_SETTING`
+
```

Comment:
Remove this step. This is out of place and was missed on previous reviews. Line 184 has the user save the workspace-credentials secret, then we go on in Line 214 to add the workspace-settings secret. We are adding both environment variables in line 218-226 as part of Step 3: Add the secrets to your container

```suggestion
```